.gitignore

```diff
@@ -27,3 +27,5 @@ build
 run
 .DS_Store
 Thumbs.db
+
+buildSrc/
```

Comment:
buildSrc shouldn't be ignored, custom build logic could be added there in the future.